scripts/install-site-in-isolation.mjs

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env zx
+
+/* eslint-disable turbo/no-undeclared-env-vars */
+import { cp, mkdir, mkdtemp } from 'node:fs/promises';
+import { tmpdir } from 'node:os';
+import { join } from 'node:path';
+
+import * as core from '@actions/core';
+import { $, argv, cd, chalk } from 'zx';
+
+try {
+  // Setup variables and environment variables
+  const SITE_PATH = argv._[0];
+  const ROOT_PATH = new URL('../', import.meta.url).pathname;
+  const FULL_SITE_PATH = join(ROOT_PATH, SITE_PATH);
+  const TMP_FOLDER = await mkdtemp(join(tmpdir(), 'clerk-site-'));
+  const FULL_TMP_FOLDER = join(TMP_FOLDER, SITE_PATH);
+  process.env.SECCO_SOURCE_PATH = ROOT_PATH;
+  process.env.FORCE_COLOR = '1';
+
+  core.debug(`Path variables:
+
+SITE_PATH: ${SITE_PATH}
+ROOT_PATH: ${ROOT_PATH}
+FULL_SITE_PATH: ${FULL_SITE_PATH}
+TMP_FOLDER: ${TMP_FOLDER}
+FULL_TMP_FOLDER: ${FULL_TMP_FOLDER}`);
+
+  // Installing secco
+  await core.group('Installing secco (if not already installed)', async () => {
+    await $`command -v secco || (command -v sudo && sudo npm install -g secco@latest) || npm install -g secco@latest`;
```

Comment:
❓ why the need for `sudo` here?

Reply:
If you run this locally (which it is designed for, you can fully run this locally) this is sometimes necessary in some environments. Ideally you already have it installed globally anyways then